src/services/tx/safeUpdateParams.ts

```diff
@@ -14,13 +44,17 @@ export const CHANGE_FALLBACK_HANDLER_ABI = 'function setFallbackHandler(address
  */
 export const createUpdateSafeTxs = (safe: SafeInfo, chain: ChainInfo): MetaTransactionData[] => {
   const latestMasterCopy = getGnosisSafeContractInstance(chain, LATEST_SAFE_VERSION)
-  const safeContractInstance = getGnosisSafeContractInstance(chain, safe.version)
+  const safeContractInterface = getContractInterface(chain, safe.version)
 
   // @ts-expect-error this was removed in 1.3.0 but we need to support it for older safe versions
-  const changeMasterCopyCallData = safeContractInstance.encode('changeMasterCopy', [latestMasterCopy.getAddress()])
+  const changeMasterCopyCallData = safeContractInterface.encodeFunctionData('changeMasterCopy', [
```

Comment:
Can we also call `.encodeFunctionData` if the contract is returned from the core-sdk? Previously it was changed to just `.encode`

Reply:
Via the `interface`, yes.